src/modinv32_impl.h

```diff
@@ -21,6 +18,14 @@
  */
 
 #ifdef VERIFY
+/* Helper function to compute the absolute value of an int32_t.
+ * (We don't use abs/labs/llabs as they depend on the int sizes and require stdlib.h.) */
+static int64_t secp256k1_modinv32_abs(int32_t v) {
+    VERIFY_CHECK(v > INT32_MIN);
+    if (v < 0) return -v;
+    return v;
+}
+
```

Comment:
This seems currently unused. Shouldn't we replace the use of `labs` below with this?

Reply:
fixed